DLM frozen tier docs (#6530)
elastic/docs-content: manage-data/data-store/data-streams/set-up-tsds.md

In most cases, you can use a [data stream lifecycle](/manage-data/lifecycle/data-stream.md) to manage your time series data stream. If you're using [data tiers](/manage-data/lifecycle/data-tiers.md) in {{stack}}, you can use [index lifecycle management](/manage-data/lifecycle/index-lifecycle-management.md).

::::{note}

> :applies_to: {"stack": "ga 9.5", "serverless": "unavailable"}

Reviewer: procedure is stack only so we can skip the `"serverless": "unavailable"` here

```suggestion
:applies_to: {"stack": "ga 9.5"}
```

> If you standardize on data stream lifecycle but still want frozen-tier {{search-snaps}}, configure `frozen_after` instead of only using {{ilm-init}} for the frozen phase. Refer to [](/manage-data/lifecycle/data-stream/frozen-searchable-snapshots.md).

Reviewer: consider:

```suggestion
{{ilm-init}} isn't required for frozen-tier {{search-snaps}}. Data stream lifecycle can manage them directly. Refer to [](/manage-data/lifecycle/data-stream/frozen-searchable-snapshots.md).
```
